vep/src/main/resources/listVariants.py

```diff
@@ -72,10 +76,11 @@ def main():
         df.varId,
     )
 
-    # output the variants as CSV part files
-    df.write \
-        .mode('overwrite') \
-        .csv(outdir, sep='\t')
+    # if we find new variants, put them in the new_variants path for additional processing
+    # and append them to the existing deduped variants path
+    if not df.rdd.isEmpty():
```

Comment:
What would happen if this were removed? If an empty df is appended to something would it just have an extra empty file?

Reply:
Yeah, it will just add an empty file(s) to both places.